CHANGELOG.md

```diff
@@ -2,6 +2,76 @@
 
 ## Unreleased
```

Comment:
Do you want to get the version in here, or does that happen at another phase?

Reply:
ya, this happens when we run the workflow to create the release branch